### PR TITLE
added user event messages

### PR DESCRIPTION
**Pull request by bookshelfdave**
See John C's Riak Event Logging gist
https://gist.github.com/jcapricebasho/4b98b1afc6204ce9df52

@bowrocker we'll need to get your _Handle decommissioned clusterid for replicated CS manifests_ commands updated as well.

@lordnull we'll need to get the _Reduced N_ commands updated to display log messages

### CODE REVIEW

File: src/riak_repl_console.erl [hunk 1]
```diff
@@ -403,26 +412,33 @@ realtime([Cmd]) ->
...
         "start" ->
+            ?LOG_USER_CMD("Start Realtime Replication to all connected clusters", []),
             [riak_repl2_rt:start(Remote) || Remote <- Remotes];
         "stop" ->
+            ?LOG_USER_CMD("Stop Realtime Replication to all connected clusters",
+                      []),
             [riak_repl2_rt:stop(Remote) || Remote <- Remotes]
     end,
     ok. %% TODO: we could gather the return codes of the list comprehensions
+        %% TODO: we could gather the return codes of the list comprehensions
```
Comment (cmeiklejohn) [inline src/riak_repl_console.erl:423]: Why'd you remove the `ok` here?

Comment (bookshelfdave) [inline src/riak_repl_console.erl:423]: it hasn't been removed, but it doesn't look like my patch worked right

Comment (cmeiklejohn) [inline src/riak_repl_console.erl:423]: Eh?

**Comment by bookshelfdave** [comment]
closed in favor of https://github.com/basho/riak_repl/pull/480

**Pull request closed without merging**